### DISCUSSION TITLE
MAPI_Error when trying to view rules table - latest version 21.0.21127 x64

### DISCUSSION THREAD

**Discussion by smithyd2003** [microsoft/mfcmapi#544] [Q&A]
Installed and running the version 21.0.21127 x64. 
Cant load rules tables - when attempting this I see the two errors below. 
I have checked and one response literally just states 'no repro.' I need a bit more guidance with this please. I did also see a link to a bug fix which showed 4 files with updates but this was quite difficult for me also. Is there not just 4 new files with the bad code removed and replaced i could get from somewhere to overwrite on my machine or do we need to code through the code and pick out the lines in question. Any help greatly appreciated!

**Error**: 
Code: MAPI_E_INTERFACE_NOT_SUPPORTED == 0x80004002
Function lpMAPIProp->OpenProperty( ulPropTag, const_cast<LPGUID>(&IID_IExchangeModifyTable), 0, MAPI_DEFERRED_ERRORS, reinterpret_cast<LPUNKNOWN*>(&lpExchTbl)) File D:\a\1\s\UI\Dialogs\MFCUtilityFunctions.cpp  Line 329

**Error**:
Code: MAPI_E_INTERFACE_NOT_SUPPORTED == 0x80004002
Function DisplayExchangeTable(lpMAPIFolder, PR_RULES_TABLE, objectType::rules, this)  File D:\a\1\s\UI\Dialogs\HierarchyTable\MsgStoreDlg.cpp  Line 227

**Comment by davenkc** [comment]
smithyd2003, did you ever resolve this issue?  I'm getting the same error message.

**Comment by stephenegriffin** [comment]
Do you repro on the [latest release](https://github.com/stephenegriffin/mfcmapi/releases)? The build you're using is two years old.
